docs/intro/quickstart/index.rst

```diff
@@ -0,0 +1,795 @@
+      import { client } from "@/lib/gel";
+
+      interface GetDeckSuccessResponse {
+        id: string;
+        name: string;
+        description: string | null;
+        cards: {
+          id: string;
+          front: string;
+          back: string;
+        }[];
+      }
+
+      interface GetDeckErrorResponse {
+        error: string;
+      }
+
+      type GetDeckResponse = GetDeckSuccessResponse | GetDeckErrorResponse;
+
+      export async function GET(
+        req: NextRequest,
+        { params }: { params: Promise<{ id: string }> }
+      ): Promise<NextResponse<GetDeckSuccessResponse | GetDeckErrorResponse>> {
+        const { id: deckId } = await params;
+        const deck = await client.querySingle<GetDeckResponse>(
+          `
+            with deckId := <uuid>$deckId,
+            select Deck {
+              id,
+              name,
+              description,
+              cards := (select .<deck[is Card] {
+                id,
+                front,
+                back,
+              } order by .order),
+            } filter .id = deckId
+          `,
+          { deckId }
+        );
+
+        if (!deck) {
+          return NextResponse.json(
+            { error: `Deck (${deckId}) not found` },
+            { status: 404 }
+          );
+        }
+
+        return NextResponse.json(deck);
+      }
+
+.. edb:split-section::
+
+  Now we can fetch the deck we created earlier by referencing its ID in the URL.
+
+  .. code-block:: sh
+
+      $ curl http://localhost:3000/api/deck/123e4567-e89b-12d3-a456-426614173000
+      {
+        "id": "123e4567-e89b-12d3-a456-426614173000",
+        "name": "Geography",
+        "description": "Questions about countries, cities, and other geographical features.",
+        "cards": [
+          {
+            "id": "123e4567-e89b-12d3-a456-426614174000",
+            "front": "What is the tallest mountain on Earth?",
+            "back": "Mount Everest"
+          },
+          {
+            "id": "123e4567-e89b-12d3-a456-426614174001",
+            "front": "What is the deepest trench on Earth?",
+            "back": "The Mariana Trench"
+          },
+          {
+            "id": "123e4567-e89b-12d3-a456-426614174002",
+            "front": "What is the widest river on Earth?",
+            "back": "The Amazon River"
+          },
+          {
+            "id": "123e4567-e89b-12d3-a456-426614174003",
+            "front": "What is the largest ocean on Earth?",
+            "back": "The Pacific Ocean"
+          },
+          {
+            "id": "123e4567-e89b-12d3-a456-426614174004",
+            "front": "What is the highest freshwater lake on Earth?",
+            "back": "Lake Titicaca"
+          }
+        ]
+      }
+
+.. edb:split-section::
+
+  As time goes on, and our planet changes, perhaps we'll want to update one of the cards with the latest in geographical knowledge. Let's add a route to update a card by its ID.
```

Comment:
I think making an "add a card to the deck" route is more interesting than updating an existing card.